api/src/wfl/log.clj

```diff
@@ -0,0 +1,84 @@
+(ns wfl.log
+  "Logging for WFL"
```

Comment:
Would be nice to specify that the logging is backed by Google Stackdriver (unless we're supporting different kinds of loggers?) along with a link to external docs.

Reply:
You are right. I do need to update with more documentation for this.